src/lib/blocks.js

```diff
@@ -5,7 +5,7 @@
  * @return {ScratchBlocks} ScratchBlocks connected with the vm
  */
 export default function (vm, useCatBlocks) {
-    const ScratchBlocks = useCatBlocks ? require('cat-blocks') : require('scratch-blocks');
+    const {ScratchBlocks} = useCatBlocks ? require('cat-blocks') : require('scratch-blocks');
```

Comment:
Uhh do we need to migrate the April Fools blocks as well? Looks like they're [hard-locked at a prerelease](https://github.com/gonfunko/scratch-gui/blob/db4b7a0ad9fbb83941b86e17e58e3df3bb48f844/package.json#L46)? Kinda confused by this because I believe this means it will be missing updates they have (like the new high contrast colors).

Reply:
Maaaybe? Filed #2 to track.